Gemfile.lock

```diff
@@ -296,10 +297,10 @@ GEM
       faraday (>= 0.17.5, < 3.a)
       jwt (>= 1.5, < 3.0)
       multi_json (~> 1.10)
-    simctl (1.6.8)
+    simctl (1.6.10)
       CFPropertyList
       naturally
-    sqlite3 (1.4.4)
+    sqlite3 (1.6.0-arm64-darwin)
```

Comment:
hmmm this is making it fail 🤔

Reply:
Ok, I had to change the platforms and I think it's now fixed:

```
bundle lock --remove-platform arm64-darwin-21
bundle lock --add-platform arm64-darwin
```

This way bundle doesn't specify this version for sqlite3

Reply:
hmm it's still failing 🤔

Reply:
I will remove the new platform for now, because it adds sqlite3 (1.6.0-arm64-darwin) which makes CircleCI fail